es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,53 @@
+import TodoList from './todo-list';
+import TodoForm from './todo-form';
+const { createElement, noop } = require('../lib');
```

Comment:
This needs to be converted to an `import` statement. When importing "named" exports you use `{ }`, it would look something like:

```
import { someThing, anotherThing } from '../some/place';
```